packages/tailwind-theme/src/plugin.ts

```diff
@@ -242,28 +242,43 @@ export default plugin(function ({ addComponents, addBase }) {
   addComponents({
     // Fancy gradient text
     '.text-fancy-gradient': {
-      '@apply font-bold bg-gradient-to-r from-blue-500 via-blue-400 to-blue-600 text-transparent bg-clip-text':
+      '@apply font-semibold bg-gradient-to-r from-blue-500 via-blue-400 to-blue-600 text-transparent bg-clip-text':
         {}
     },
-    // Font sizes
-    '.h1': {
-      '@apply text-5xl leading-10': {}
+    // New heading styles
+    '.text-heading-2xl, .h1': {
+      '@apply font-medium text-3xl leading-10 tracking-[-0.63px]': {}
```

Comment:
`tracking-[-0.63px]` I guess this is because we are converting % from Figma to px values?

Reply:
Yeah, these pixel values were provided by Michal. Maybe one to come back to in future if we experience issues around browser base font changes.